go/libraries/doltcore/sqle/system_variables.go

```diff
@@ -256,6 +256,245 @@ var DoltSystemVariables = []sql.SystemVariable{
 }
 
 func AddDoltSystemVariables() {
+	sql.SystemVariables.AddSystemVariables([]sql.SystemVariable{
```

Comment:
Would rather move these into the named var below (also seems like there is duplication between these two lists?